scripts/presets/travis/sharded_clusters/cluster_replset.json

```diff
@@ -0,0 +1,117 @@
+{
+    "configsvrs": [ {
+        "members" : [
+            {
+                "procParams": {
+                    "dbpath": "/tmp/SHARDED-RS/CFG/4490",
+                    "logpath": "/tmp/SHARDED-RS/CFG/4490/mongod.log",
+                    "ipv6": true,
+                    "journal": true,
+                    "logappend": true,
+                    "nssize": 1,
+                    "port": 4490,
+                    "bind_ip_all": true,
+                    "smallfiles": true
+                }
+            },
+            {
+                "procParams": {
+                    "dbpath": "/tmp/SHARDED-RS/CFG/4491",
+                    "logpath": "/tmp/SHARDED-RS/CFG/4491/mongod.log",
+                    "ipv6": true,
+                    "journal": true,
+                    "logappend": true,
+                    "nssize": 1,
+                    "port": 4491,
+                    "bind_ip_all": true,
+                    "smallfiles": true
+                }
+            },
+            {
+                "procParams": {
+                    "dbpath": "/tmp/SHARDED-RS/CFG/4492",
+                    "logpath": "/tmp/SHARDED-RS/CFG/4492/mongod.log",
+                    "ipv6": true,
+                    "journal": true,
+                    "logappend": true,
+                    "nssize": 1,
+                    "port": 4492,
+                    "bind_ip_all": true,
+                    "smallfiles": true
+                }
+            }
+        ]
+    } ],
+    "id": "cluster_2", 
```

Comment:
Would "cluster_rs" make more sense, given the IDs you use below? Either way, I assume this name isn't important as we don't reference it in URIs.

Reply:
Doesn't matter, but I'll change it.